libcaf_core/caf/event_based_actor.test.cpp

```diff
@@ -19,7 +19,7 @@ WITH_FIXTURE(test::fixture::deterministic) {
 TEST("unexpected messages result in an error by default") {
   auto receiver = sys.spawn([](event_based_actor*) -> behavior {
     return {
-      [](int32_t) {},
+      [](int32_t x) { return ++x; },
```

Comment:
Why mutate the variable first if we can just return the result? IMHO cleaner this way:

```suggestion
      [](int32_t x) { return x + 1; },
```